### PR TITLE
Remove adData

### PR DESCRIPTION
**Pull request by mariogalic**
https://trello.com/c/eLPxVEIC/912-remove-addata

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/user-ad-targeting.spec.js [hunk 1]
```diff
@@ -1,21 +1,13 @@
...
 
-const getUserFromApi: any = getUserFromApi_;
 const getUserFromCookie: any = getUserFromCookie_;
 
 jest.mock('lib/storage');
 jest.mock('common/modules/identity/api', () => ({
     getUserFromCookie: jest.fn(),
-    getUserFromApi: jest.fn(),
 }));
 const userSegmentsKey = 'gu.ads.userSegmentsData';
```
Comment (jfsoul) [inline static/src/javascripts/projects/common/modules/commercial/user-ad-targeting.spec.js]: Are we able to check with @guardian/commercial-dev whether this is used any more?

**Comment by jfsoul** [comment]
cc: @jeteve 

for background we have some legacy data stored in Identity (inserted by Acxiom) that appears to get inserted into local storage.  Want to double check with you whether this is used as far as you know?

**Comment by jeteve** [comment]
Hi all @jfsoul @mario-galic ,

If you're positive there was no hit on the endpoint over the last week, then I don't have anything about removing it.

One question: the card mentions an endpoint '/crm/user/' , but the 'getUserFromApi' function seems to be using something that ends with 'user/me' . Are you sure those are the same things?

Cheers,

J.

**Comment by jeteve** [comment]
@mario-galic I'd also wait for @ScottPainterGNM  to be back next week just to be on the safe side, as there's maybe a risk for this to affect revenue.

**Comment by mario-galic** [comment]
> One question: the card mentions an endpoint '/crm/user/' , but the 'getUserFromApi' function seems to be using something that ends with 'user/me' . Are you sure those are the same things?

No longer used system would post `adData` into `users` table in Identity via `/crm/user` endpoint, which would then be read via `user/me`.

**Comment by mchv** [comment]
@mario-galic May you please change the description this PR to include the content of the ticket rather than link? Thanks.

**Pull request closed without merging**